proto/api.proto

```diff
@@ -1828,6 +1850,41 @@ message GetJupiterPricesResponse {
   repeated TokenPriceV2 tokenPrices = 1;
 }
 
+message GetZetaTransactionStreamRequest {
+  repeated string instructions = 1;
+}
+
+message Transaction {
+  repeated string signatures = 1;
+  TransactionMessageZeta message = 2;
+}
+
+message TransactionMessageZeta {
+  TransactionMessageHeader header = 2;
+  repeated string account_keys = 3;
+  string recent_blockhash = 4;
+  repeated Instruction instructions = 5;
+  repeated AddressTableLookup address_table_lookups = 6;
+}
+
+message TransactionMessageHeader {
+  uint32 num_required_signatures = 1;
+  uint32 num_readonly_signed_accounts = 2;
+  uint32 num_readonly_unsigned_accounts = 3;
+}
+
+message AddressTableLookup {
```

Comment:
we can skip the lookup table since Geyser transactions come with an  AccountsKeys slice that already contains TableLookup accounts...so this is just extra weight. Unless this is needed for PostZetaCrossMarginAccountResponse

Reply:
removed